documentation/configuration.md

```diff
@@ -759,6 +759,104 @@ Usage that executes a .NET executable named "myapp.dll" using `dotnet`.
 }
 ```
 
+#### `LoadProfiler` Action
+
+An action that loads an ICorProfilerCallback implementation into a target process as a startup profiler. This action must be used in a collection rule with a `Startup` trigger.
+
+##### Properties
+
+| Name | Type | Required | Description | Default Value |
+|---|---|---|---|---|
+| `Path` | string | true | The path of the profiler library to be loaded. This is typically the same value that would be set as the CORECLR_PROFILER_PATH environment variable. | |
+| `Clsid` | Guid | true | The class identifier (or CLSID, typically a GUID) of the ICorProfilerCallback implementation. This is typically the same value that would be set as the CORECLR_PROFILER environment variable. | |
+
+##### Outputs
+
+No outputs
+
+##### Example
+
+Usage that loads one of the sample profilers from [`dotnet/runtime`: src/tests/profiler/native/gcallocateprofiler/gcallocateprofiler.cpp](https://github.com/dotnet/runtime/blob/9ddd58a58d14a7bec5ed6eb777c6703c48aca15d/src/tests/profiler/native/gcallocateprofiler/gcallocateprofiler.cpp).
+
+```json
+{
+  "Path": "Profilers\\Profiler.dll",
+  "Clsid": "55b9554d-6115-45a2-be1e-c80f7fa35369"
+}
+```
+
+#### `SetEnvironmentVariable` Action
+
+An action that sets an environment variable value in the target process. This action should be used in a collection rule with a `Startup` trigger.
+
+##### Properties
+
+| Name | Type | Required | Description | Default Value |
+|---|---|---|---|---|
+| `Name` | string | true | The name of the environment variable to set. | |
```

Comment:
Have we considered the ability to set multiple environment variables with one action? For example, if i wanted to add clrie here i would have _a lot_ of actions

Reply:
The answer is to use many actions. I was thinking it'll be overkill for most people that need to set 2 variables.

Reply:
This is something we can add in the future and dynamically determine during options binding if we really need it.

Reply:
Exactly, reducing the config size would be the only benefit, which would be meaningful, but we can do that later if needed.